src/Zir.zig

```diff
@@ -248,6 +248,10 @@ pub const Inst = struct {
         /// Given a pointer type, returns its element type.
         /// Uses the `un_node` field.
         elem_type,
+        /// Given a vector type, returns its element type.
```

Comment:
After applying change suggested in previous comment, this should then probably also mention something like `/// Resolves the type as a destination type from RLS, i.e. emitting an error if it is unknown due to a generic parameter.`